provider-blackduck/src/main/java/com/synopsys/integration/alert/provider/blackduck/processor/message/service/BlackDuckMessageBomComponentDetailsCreator.java

```diff
@@ -102,8 +106,10 @@ public BomComponentDetails createBomComponentDetails(ProjectVersionComponentVers
         );
     }
 
-    public BomComponentDetails createBomComponentUnknownVersionDetails(ProjectVersionComponentVersionView bomComponent, List<ComponentConcern> componentConcerns, ComponentUpgradeGuidance componentUpgradeGuidance,
-        List<LinkableItem> additionalAttributes) throws IntegrationException {
+    public BomComponentDetails createBomComponentUnknownVersionDetails(
+        ProjectVersionComponentVersionView bomComponent, List<ComponentConcern> componentConcerns, ComponentUpgradeGuidance componentUpgradeGuidance,
+        List<LinkableItem> additionalAttributes
+    ) throws IntegrationException {
```

Comment:
Should this be on a new line?
`) throws IntegrationException {`

Reply:
made the change